packages/extension-mention/src/mention.ts

```diff
@@ -1,11 +1,14 @@
 import { mergeAttributes, Node } from '@tiptap/core'
-import { Node as ProseMirrorNode } from '@tiptap/pm/model'
+import { DOMOutputSpec, Node as ProseMirrorNode } from '@tiptap/pm/model'
 import { PluginKey } from '@tiptap/pm/state'
 import Suggestion, { SuggestionOptions } from '@tiptap/suggestion'
 
 export type MentionOptions = {
   HTMLAttributes: Record<string, any>
+  /** @deprecated use renderText and renderHTML instead  */
   renderLabel: (props: { options: MentionOptions; node: ProseMirrorNode }) => string
+  renderText: (props: { options: MentionOptions; node: ProseMirrorNode }) => string
```

Comment:
What's the point of adding this?

Reply:
Yo mean renaming renderLabel to renderText? This was done to be consistent with other extensions